olp-cpp-sdk-core/src/cache/DiskCache.cpp

```diff
@@ -52,7 +52,7 @@ static bool RepairCache(const std::string& data_path) {
   auto status = leveldb::RepairDB(data_path, leveldb::Options());
   if (status.ok()) {
     OLP_SDK_LOG_INFO(kLogTag, "RepairCache: repaired - " << data_path);
-    DiskCacheEnv::Env()->DeleteDir(data_path + "/lost");
+    utils::Dir::Remove(data_path + "/lost");
```

Comment:
Please move "lost/" to a constexpr.
I would not let this silently go. Instead, please log that there was a lost folder and we will delete it.
```cpp
if (utils::Dir::Exists(data_path + kLeveldbLostFolder)) {
    OLP_SDK_LOG_WARNING(kLogTag, "RepairCache: data has been lost - deleting '" << kLeveldbLostFolder << "' folder");
    utils::Dir::Remove(data_path + "/lost");
}
```

Reply:
I'm not sure about the 'data has been lost' part. This may be misleading as it's not guaranteed that any data was lost and vice versa.

Reply:
Then "RepairCache: data may have been lost ...."